app/collections/show-list.cjsx

```diff
@@ -140,14 +184,35 @@ module.exports = React.createClass
           meta = @state.subjects[0].getMeta()
 
           <div>
-            <div className="collection-subjects-list">
+          {if @state.selecting
```

Comment:
or maybe disable the Select Subjects button if signed-out?
or maybe a better question is what behavior is expected if signed-out? if I make the Add to Collection button not visible to a signed-out user, they can still click the Select Subjects button and the circles show, but it's not clear what's going on.